docs/src/content/docs/utilities/add-virtual-imports.mdx

```diff
@@ -36,14 +43,26 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
         return {
             name: "my-integration",
             hooks: {
-                "astro:config:setup": ({ updateConfig }) => {
+                "astro:config:setup": ({ config, updateConfig }) => {
                     addVirtualImports({
                         updateConfig,
                         name: 'my-integration',
                         imports: {
                             'virtual:my-integration/config': `export default ${JSON.stringify({ foo: "bar" })}`,
                         },
                     })
+                    addVirtualImports({
+                        config,
+                        updateConfig,
+                        name: 'my-integration',
+                        imports: [
+                            {
+                                id: "virtual:my-integration/server-only",
+                                content: "export const foo = 'bar'",
+                                context: "server"
+                            }
+                        ]
```

Comment:
There could be a client import here to show that you can have the same import in different contexts